src/overlays/actors/ovl_En_Ma4/z_en_ma4.h

```diff
@@ -2,15 +2,37 @@
 #define Z_EN_MA4_H
 
 #include <global.h>
+#include "overlays/actors/ovl_En_Ma_Yts/z_en_ma_yts.h"
 
 struct EnMa4;
 
+typedef void (*EnMa4ActionFunc)(struct EnMa4*, GlobalContext*);
+
 typedef struct EnMa4 {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x1F4];
-    /* 0x338 */ s16 unk338; // modified by EnPoFusen
-    /* 0x33A */ char unk33A[0x6];
-    
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ EnMa4ActionFunc actionFunc;
+    /* 0x18C */ ColliderCylinder collider;
+    /* 0x1D8 */ struct_800BD888_arg1 unk_1D8;
+    /* 0x200 */ Vec3s* pathPoints;
+    /* 0x204 */ Vec3s limbDrawTable[MA1_LIMB_MAX];
+    /* 0x28E */ char unk28E[0x6];
+    /* 0x294 */ Vec3s transitionDrawTable[MA1_LIMB_MAX];
+    /* 0x31E */ char unk31E[0x6];
```

Comment:
```suggestion
    /* 0x31E */ UNK_TYPE1 unk31E[0x6];
```

Reply:
A bit offtopic, but I think this dev somehow messed up Romani's limb enum and added an extra limb. `EnMaYts` has this exact same "problem".